semgrep-core/matching/Generic_vs_generic.ml

```diff
@@ -1085,6 +1085,19 @@ and m_list__m_argument (xsa: A.argument list) (xsb: A.argument list) =
   | [A.Arg (A.Ellipsis _i)], [] ->
       return ()
 
+  | A.Arg (A.Id ((s, tok), _idinfo))::xsa, xb::xsb
+    when MV.is_metavar_ellipsis s ->
+      (* can match 1 or more arguments (or 0 is ok too?) *)
```

Comment:
My intuition is that we could match 0 items, but this might lead to weird matching in edge cases...

E.g., I'd expect to be able to match:
```python
unsafe_api_call($ARGS...)
```

to
```python
unsafe_api_call()
```

so I could have an autofix:
```python
safe_api_call(", ".join($ARGS))
```

Reply:
I was just scared that if it matches nothing, then we will get some NoTokenLocation errors.
But yes my intuition is that I think also it should match 0 or more.